### DISCUSSION TITLE
Litecoin no longer works properly

### DISCUSSION THREAD

**Discussion by Devyre** [btcpayserver/btcpayserver#4626] [Technical Support]
Hello,

I've been using BTCPay Server for almost 2 years now, never updated, and suddenly when sending crypto I had HTTP Error 500 happen 90% of the time. That was around a week ago.
So I thought "well, let me try to update and see if it persists". So I did. The http error was gone.
But ever since the update, LTC is "dead". Customers can create an invoice, but once they pay, the money will not shop up as paid. I have transactions to back up that it was in fact paid. When I search the LTC wallet address the money was sent to, it shows the correct invoice.

Do I need to manually enable litecoin again? It's displayed in the web interface and is green just like BTC so I'd assume it's already enabled / supposed to work.

**Comment by mp21-org** [comment]
Hi @Devyre 

we had a couple of users with this issue that when updating Litecoind would suddenly stop syncing. You can check your logs with:
`docker logs btcpayserver_litecoind`

You will probably see errors beginning with "AcceptBlockHeader"

This means that your local blockchain copy is corrupted and a resync is needed.

Nicolas wrote down the needed steps in our [Mattermost](https://chat.btcpayserver.org/btcpayserver/channels/support/o58a6qa5ibrsf8i18yt4g5yuac):

```
btcpay-down.sh
rm -rf /var/lib/docker/volumes/generated_litecoin_datadir/_data/blocks
rm -rf /var/lib/docker/volumes/generated_litecoin_datadir/_data/chainstate
btcpay-up.sh
```

some get a device is busy error in which case, the advise is to do:
maybe `rm -rf /var/lib/docker/volumes/generated_litecoin_datadir/_data/blocks/*` is better

Hope this gets you running again. If you have further questions, please drop by on Mattermost. Github discussions aren't great for support questions.
